cg/constants/housekeeper_tags.py

```diff
@@ -76,9 +76,9 @@ class HkMipAnalysisTag:
     SAMPLE_INFO: list[str] = ["sample-info"]
 
 
-class BalsamicAnalysisTag(StrEnum):
-    CONFIG: str = "balsamic-config"
-    QC_METRICS: str = "qc-metrics"
+class BalsamicAnalysisTag:
```

Comment:
it could be an `Enum` class as well

Reply:
```suggestion
class BalsamicAnalysisTags:
```

Reply:
I'll keep it as is to align with the naming of the other pipeline analysis tags. Thank you for the review!